File: app/src/main/java/com/jedev/leflix/RegisterActivity.java
package com.jedev.leflix;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

import com.google.firebase.auth.FirebaseAuth;
import com.jedev.leflix.Model.Usuario;

public class RegisterActivity extends AppCompatActivity {
    private FirebaseAuth mAuth;
    private EditText campoNome, campoEmail, campoSenha;
    private Button botaoCadastrar;

    private Usuario usuario;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.register_main);
        campoNome = findViewById(R.id.regsenha);
        campoEmail = findViewById(R.id.regemail);
        campoSenha = findViewById(R.id.regsenha);
        botaoCadastrar = findViewById(R.id.regcadastrar);



    }
}

File: app/src/main/java/com/jedev/leflix/view/RegisterActivity.kt
package com.jedev.leflix.view

import android.os.Bundle

import androidx.appcompat.app.AppCompatActivity

import com.jedev.leflix.R
import com.jedev.leflix.controllers.RegisterController
import kotlinx.android.synthetic.main.activity_register.*

class RegisterActivity : AppCompatActivity() {

    val controller = RegisterController(this)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_register)

        btn_login.setOnClickListener {
            controller.openLogin()
        }

        btn_register.setOnClickListener {
            controller.onRegister(edt_name, edt_email, edt_password)
        }
    }


}
